src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -39,31 +40,66 @@
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
-		<!--  Titlebar  -->
-		<Border x:Name="TitlebarArea" Height="40">
+		<!--  Titlebar Area  -->
+		<Grid x:Name="TitlebarAreaGrid">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="Auto" />
+				<ColumnDefinition Width="*" />
+			</Grid.ColumnDefinitions>
+
+			<Button
+				x:Name="TitlebarAreaBackButton"
+				Width="40"
+				Height="32"
+				Margin="4,4,0,0"
+				Background="Transparent"
+				BorderThickness="0"
+				Click="TitlebarAreaBackButton_Click"
+				IsEnabled="{x:Bind MainContentFrame.CanGoBack, Mode=OneWay}">
+				<Button.Resources>
+					<ResourceDictionary>
+						<StaticResource x:Key="ButtonBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+					</ResourceDictionary>
+				</Button.Resources>
+				<AnimatedIcon x:Name="BackAnimatedIcon">
+					<AnimatedIcon.Source>
+						<animatedvisuals:AnimatedBackVisualSource />
+					</AnimatedIcon.Source>
+					<AnimatedIcon.FallbackIconSource>
+						<SymbolIconSource Symbol="Back" />
+					</AnimatedIcon.FallbackIconSource>
+				</AnimatedIcon>
+			</Button>
+
 			<TextBlock
-				Padding="16,0"
+				x:Name="TitlebarAreaTitle"
+				Grid.Column="1"
+				Margin="4,0,4,-2"
```

Comment:
Why is there a negative margin?

Reply:
Because I needed to adjust TextBlock placement with Back Button Glyph center. If not, it looks a bit strange

Reply:
Can you show a screenshot?